### PR TITLE
[MVC-988] Join-Us page Job Listing section fixes

### PR DESCRIPTION
**Pull request by SydneyMhashu**
Join us page changes
1) To display job listings in 3 rows
2) Overflow all excess listings
3) Adding a horizontal scroller
4) Buttons to scroll left and right horizontally

### CODE REVIEW

**Comment by andregdias** [comment]
Screenshots?

File: _includes/jobs.html [hunk 1]
```diff
@@ -5,14 +5,37 @@ <h1>Open positions</h1>
...
-        <p>{{job.['Job Description']}}</p>
-        <small><a href="{{job.Link}}">Read full description</a></small>
+        <div class="arrows__container">
+         <svg width="129" height="129" class="svg__arrow_1" id="svg__arrow_1" viewBox="0 0 129 129" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M129 64.5c0 35.622-28.878 64.5-64.5 64.5S0 100.122 0 64.5 28.878 0 64.5 0 129 28.878 129 64.5z" fill="#2C449C"/><path fill-rule="evenodd" clip-rule="evenodd" d="M96.82 68.544L68.182 97.182a4.5 4.5 0 01-6.364 0L33.18 68.544a4.5 4.5 0 116.364-6.364L60.5 83.136V33h9v50.136L90.456 62.18a4.5 4.5 0 116.364 6.364z" fill="#FDEBCB"/></svg>
+         <svg width="129" class="svg__arrow_2" id="svg__arrow_2" height="129" viewBox="0 0 129 129" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M129 64.5c0 35.622-28.878 64.5-64.5 64.5S0 100.122 0 64.5 28.878 0 64.5 0 129 28.878 129 64.5z" fill="#2C449C"/><path fill-rule="evenodd" clip-rule="evenodd" d="M96.82 68.544L68.182 97.182a4.5 4.5 0 01-6.364 0L33.18 68.544a4.5 4.5 0 116.364-6.364L60.5 83.136V33h9v50.136L90.456 62.18a4.5 4.5 0 116.364 6.364z" fill="#FDEBCB"/></svg>
+       </div>
```
Comment (raguiar9080) [inline _includes/jobs.html:11]: indentation here seems off. should be a bit more to the left, to match previous and posterior indentation

File: _includes/jobs.html [hunk 2]
```diff
@@ -5,14 +5,37 @@ <h1>Open positions</h1>
...
+          </div>
       </div>
-      {% endfor %}
     </div>
   </div>
 </div>
+
+
+<script>
+  console.log('Built by sida');
```
Comment (raguiar9080) [inline _includes/jobs.html:30]: Is this necessary?

File: assets/main.js [hunk 3]
```diff
@@ -273,6 +273,26 @@ function filters(){
...
+//     console.log('Hello');
+// }
+
+// $(document).ready(function() {
+//   $('#pnAdvancerLeft').on('click', function() {
+//     console.log('Hello');
+//   });
+// });
+
+
```
Comment (raguiar9080) [inline assets/main.js:295]: This code is unused ✂️

**Review by raguiar9080 — COMMENTED** [review]
Left some indentation and commented code issues but looking good 👌

**Comment by tiagovarela** [comment]
@SydneyMhashu This PR includes jQuery but I'm not sure we are using it though. Should it be included like this?

**Comment by SydneyMhashu** [comment]
Hi guys
I have done the clean up. 
Looks like the whole PR will be re-worked as soon as i get new direction in tomorrows meeting so, no need to merge yet

**Pull request closed without merging**